mysql/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG - mysql
 
+1.0.4 / UNRELEASED
```

Comment:
We use `Unreleased` (the tooling it not perfect right now and expect the exact value)